3p/beopinion.js

```diff
@@ -74,9 +74,7 @@ function createContainer(global, data) {
 
   setStyles(container, {
     width: '100%',
```

Comment:
Can remove the `width: '100%'`, it is redundant with `display: block`.

Reply:
it might be better to keep it in cases it's contained in a flex element. the iframe we inject inside uses a `width: 1px; min-width: 100%` trick due to a iOS bug. 

<img width="1107" alt="Screenshot 2019-09-26 at 11 39 50" src="https://user-images.githubusercontent.com/1688645/65677717-65b7df80-e052-11e9-9217-13d39699e8e2.png">
<img width="1105" alt="Screenshot 2019-09-26 at 11 40 01" src="https://user-images.githubusercontent.com/1688645/65677718-66507600-e052-11e9-8a75-9b62ebe16d3e.png">